index.html

```diff
@@ -0,0 +1,123 @@
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <nav class="nav-bar flex-center one-edge-shadow">
+      <div class="nav-primary">
+        <span class="nav-bar-logo txt-high-light">ASIMOV</span>
+        <a href="#" class="toggle-btn">
+          <span class="toggle-bar"></span>
+          <span class="toggle-bar"></span>
+          <span class="toggle-bar"></span>
+        </a>
+      </div>
+
+      <div class="nav-bar-pills">
+        <ul class="flex-center">
+          <li><a href="#" class="nav-link">Home</a></li>
+          <li><a href="#" class="nav-link">Documentation</a></li>
+          <li><a href="#" class="nav-link">Examples</a></li>
+        </ul>
+      </div>
+    </nav>
+    <main class="section-container-50-50 container-gutter">
+      <img src="./assets/show-case.svg" class="main-img" />
+      <section class="text-wrapper holo-bg rounded-border section-gutter">
+        <h2 class="txt-high-light main-text heading-large">
+          Build for the future
+        </h2>
+        <p class="text main-text text-large">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam,
+          ipsa veniam soluta excepturi officia quod incidunt quidem, ad hic
+          labore iure, expedita illo et similique quasi eaque numquam pariatur
+          sapiente?
+        </p>
+        <div class="call-to-action">
+          <a href="#installation" class="secondary-button">Installation</a>
+          <a href="#" class="primary-button">Documentation</a>
+        </div>
+      </section>
+    </main>
+    <section id="installation" class="section-container-50-50 container-gutter">
+      <div class="text-wrapper holo-bg rounded-border section-gutter">
+        <h2 class="headings txt-high-light main-text heading-large">
+          Installation
+        </h2>
+        <p class="text main-text text-large">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam,
+          ipsa veniam soluta excepturi officia quod incidunt quidem, ad hic
+          labore iure, expedita illo et similique quasi eaque numquam pariatur
+          sapiente?
+        </p>
+      </div>
+      <div>
+        <code>
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%2835%2C138%2C224%2C0.05%29&t=one-light&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=133%25&si=false&es=2x&wm=false"
+            style="
+              width: 953px;
+              height: 320px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+            class="main-img"
+          >
+          </iframe>
```

Comment:
If possible, try to change the background of the code snippet to something darker. So that it would look even better with the overall theme of your page.